packages/rlc-common/package.json

```diff
@@ -41,7 +41,9 @@
     "@types/lodash": "^4.14.182",
     "fs-extra": "^10.0.0",
     "@types/fs-extra": "^8.1.0",
-    "ts-node": "^10.7.0"
+    "ts-node": "^10.7.0",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0"
   },
```

Comment:
Config is already there: https://github.com/Azure/autorest.typescript/blob/8c9e444c88e09311b23c6501a0dd592c52c32225/packages/rlc-common/.eslintrc.json